src/promptflow-core/promptflow/_core/tool_meta_generator.py

```diff
@@ -261,10 +261,23 @@ def load_python_module_from_file(src_file: Path):
     try:
         spec.loader.exec_module(m)
     except Exception as e:
+        message = "Failed to load python module from file '{src_file}': {error_type_and_message}"
+        if (
+            isinstance(e, ImportError)
+            and hasattr(e, "msg")
+            and e.msg == "attempted relative import with no known parent package"  # pylint: disable=no-member
+        ):
+            message = (
+                "Failed to load python module from file '{src_file}' due to relative import."
+                + " Relative imports fail in loaded code that is not a known python module."
+                + " Please use absolute imports instead (ex: 'import .utils' instead of 'import utils'),"
```

Comment:
Is this not relative import? Looks like either wording or example is incorrect.

Reply:
reworded to avoid that confusion.